components/brave_new_tab_ui/containers/newTab/sponsored_rich_media_background.tsx

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import styled from 'styled-components'
+
+export interface SponsoredRichMediaBackgroundInfo {
+  url: string
+  creativeInstanceId: string
+  placementId: string
+  targetUrl: string
+}
+
+interface StatusProps {
+  sponsoredRichMediaBackgroundHasLoaded: boolean
```

Comment:
Nit: consider whether a shorter name would be more readable (also for `sponsoredRichMediaBackgroundInfo`).

Reply:
changed to richMediaHasLoaded